### PR TITLE
new(rules): add umount macro

### PR DESCRIPTION
**Pull request by incertum**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area rules

> /area registry

> /area build

> /area documentation

**What this PR does / why we need it**:

add new `umount` macro

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -74,6 +74,11 @@
 - macro: modify
   condition: (rename or remove)
 
+# %evt.arg.flags available for evt.dir=>, but only for umount2
+# %evt.arg.name is path and available for evt.dir=<
+- macro: umount
```
Comment (loresuso) [inline rules/falco_rules.yaml:79]: is this for an upcoming rule or for addressing https://github.com/falcosecurity/falco/issues/2443#issuecomment-1471289452? in that case, do we want macros also for the event types (e.g pipe,pipe2) already put in place?

**Comment by jasondellaluce** [comment]
My only concern of adding macros that are not used in rules is that Falco will return the "XXX macro is not used" warning when processing the ruleset for validation.

**Comment by Andreagit97** [comment]
Agree with Jason here, we could comment it and leave it as a hint for users, WDYT?

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**